Resources/Prototypes/Entities/Objects/Materials/scrap.yml

```diff
@@ -0,0 +1,407 @@
+  - type: Sprite
+    layers:
+    - state: junk-firelock-2
+  - type: Item
+    size: Ginormous
+    heldPrefix: firelock-2
+  - type: MultiHandedItem
+  - type: PhysicalComposition
+    materialComposition:
+      Steel: 2000 # 20 sheets
+
+- type: entity
+  parent: BaseScrap
+  id: ScrapFirelock3
+  name: firelock frame
+  components:
+  - type: Sprite
+    layers:
+    - state: junk-firelock-3
+  - type: Item
+    size: Large
+  - type: PhysicalComposition
+    materialComposition:
+      Steel: 700 # 7 sheets
+
+- type: entity
+  parent: BaseScrap
+  id: ScrapIntercom
+  name: intercom scrap
+  description: You wish it at least came with some encryption keys.
+  components:
+  - type: Sprite
+    layers:
+    - state: junk-intercom-1
+      map: [ "base" ]
+  - type: RandomSprite
+    available:
+    - base:
+        junk-intercom-1: ""
+        junk-intercom-2: ""
+        junk-intercom-3: ""
+  - type: Item
+  - type: PhysicalComposition
+    materialComposition:
+      Steel: 400 # 4 sheets
+      Plastic: 300 # 3 sheets
+
+- type: entity
+  parent: BaseScrap
+  id: ScrapJetpack
+  name: busted jetpack
+  description: Looks like it won't be flying any more.
+  components:
+  - type: Sprite
+    layers:
+    - state: junk-jetpack
+  - type: Item
+    size: Huge
+    heldPrefix: jetpack
+  - type: PhysicalComposition
+    materialComposition:
+      Steel: 1000 # 10 sheets
+      Plastic: 200 # 2 sheets
+
+- type: entity
+  parent: BaseScrap
+  id: ScrapMedkit
+  name: snapped medkit
+  description: Hopefully they got some use out of it.
+  components:
+  - type: Sprite
+    layers:
+    - state: junk-medkit-1
+      map: [ "base" ]
+  - type: RandomSprite
+    available:
+    - base:
+        junk-medkit-1: ""
+        junk-medkit-2: ""
+  - type: Item
+    size: Large
+  - type: PhysicalComposition
+    materialComposition:
+      Plastic: 500 # 5 sheets
+
+- type: entity
+  parent: BaseScrap
+  id: ScrapMopBucket
+  name: half a mop bucket
+  description: Tiders die. Janitors survive.
+  components:
+  - type: Sprite
+    layers:
+    - state: junk-mop-bucket
+  - type: Item
+    size: Huge
+  - type: PhysicalComposition
+    materialComposition:
+      Plastic: 1500 # 15 sheets
+
+- type: entity
+  parent: BaseScrap
+  id: ScrapPAI
+  name: personal ai device
+  description: A buddy! You still in there? Hello? Buddy???
+  components:
+  - type: Sprite
+    layers:
+    - state: junk-pai
+  - type: Item
+    size: Small
+  - type: PhysicalComposition
+    materialComposition:
+      Steel: 300 # 3 sheets
+      Plastic: 200 # 2 sheets
+
+- type: entity
+  parent: BaseScrap
+  id: ScrapPAIGold
+  name: gilded personal ai device
+  description: Wow! What a one-in-a-million find! This would be worth a fortune... if it was in good condition.
+  components:
+  - type: Sprite
+    layers:
+    - state: junk-pai-gold
+  - type: Item
+    size: Small
+  - type: PhysicalComposition
+    materialComposition:
+      Plastic: 100 # 1 sheet
```

Comment:
Is this an error? This is worth less value then the other scrap pAI, which doesn't really make sense, since it seems to be made out of gold.

Reply:
You'd think it'd be made of gold